### ISSUE TITLE
Adding an Accessibility Tips page

### ISSUE THREAD

**Issue by cyberbaloo** [suitenumerique/docs#1988]
## Feature Request

**Is your feature request related to a problem or unsupported use case? Please describe.**
This Accessibility Tips&Tricks page could help reassure users about the accessibility features offered by Docs.

**Describe the solution you'd like**
A page with a list of the proposed features, a list of shortcuts implemented to access, for example, the sub-documents access panel (F2) etc... And this page would be accessible right away, even before being connected to the application, to be transparent with users with disabilities.

**Describe alternatives you've considered**
This page must be in HTML format and not created on a content management platform, which could potentially be more difficult to access with keyboard navigation, for example. I'm thinking particularly of the support section.

**Comment by Ovgodd** [comment]
I agree but this should be discussed with @robin-lecomte  and @virgile-dev

**Comment by virgile-dev** [comment]
@cyberbaloo How about we write the content on this [doc](https://docs.numerique.gouv.fr/docs/01053986-e4e2-4f5d-8ff9-ac47693222fa/?utm_source=docssharelink&utm_campaign=01053986-e4e2-4f5d-8ff9-ac47693222fa) ?
We'll need to have both versions, french and english.
Then we can see how we can add a static page on the django app.

**Comment by cyberbaloo** [comment]
@virgile-dev it's good! First, in French and English and then in the other languages offered by Docs.

**Comment by virgile-dev** [comment]
We'll add the translations strings to crowdin so they can be translated by the community.
